### DISCUSSION TITLE
Can someone provide me command to generate image

### DISCUSSION THREAD

**Discussion by xSunShine** [mgeeky/PackMyPayload#9] [Q&A]
Hello,

I had managed before to generate payload into img but now i can't do it i do the output file with .img extention and it packs into iso . i tryed all other kinda of commands nothing worked.Can someone provide me command to pack payload into .img?

**Comment by mgeeky** [comment]
Hey, will sit down to this and troubleshoot it if it's code bug. As a workaround, you can simply rename .iso to .img - its really the same format underneath the hood and that's what PackMyPayload is actually doing, just renaming byproduct. 

Lemme know if renaming was enough to get you back on track.

**Reply by xSunShine** [reply]
Hey,

Will try doing that and let you know here with update.

UPDATE: still opening as iso even if i rename it

**Reply by xSunShine** [reply]
this is my command :


`python PackMyPayload.py  /home/username/Downloads/chrome.exe /home/username/Desktop/slika3.img -v` correct me if im worng with command and this is the output: 


+      o     +              o   +      o     +              o
    +             o     +           +             o     +         +
    o  +           +        +           o  +           +          o
-_-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-_-_-_-_-_-_-_,------,      o
   :: PACK MY PAYLOAD (1.2.0)       -_-_-_-_-_-_-|   /\_/\
   for all your container cravings   -_-_-_-_-_-~|__( ^ .^)  +    +
-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-__-_-_-_-_-_-_-''  ''
+      o         o   +       o       +      o         o   +       o
+      o            +      o    ~   Mariusz Banach / mgeeky    o
o      ~     +           ~          <mb [at] binary-offensive.com>
    o           +                         o           +           +

[.] Packaging input file to output .img (iso)...
Burning file onto ISO:
    Adding file: /chrome.exe
[INFO] [+] File packaged into ISO.
[INFO] Successfully packed input file.

[+] Generated file written to (size: 75776): /home/username/Desktop/slika3.img

**Reply by mgeeky** [reply]
Unsure what's wrong here, you got your .IMG (that is actually an .ISO in its low-level), so what's wrong here?

**Reply by xSunShine** [reply]
before when i generate img with ur tool it generate a file with img when it open, now it says extention img but its iso

**Reply by mgeeky** [reply]
IMG = ISO-renamed

Sorry, I don't mean to be rude but I really don't understand what's the issue here

**Reply by xSunShine** [reply]
haha :) i want to clarify also, i want output file with. img and when open it opens picture not iso with extention img and it is iso, can you provide me example of command to create img file maybe im doing something wrong, btw i guess. img file when open it should open one of the images from img folder in packmypayload?

**Reply by xSunShine** [reply]
Here you can see when i try to generate img how file looks like , and i want to pack my exe payload into img file. not iso

![image](https://user-images.githubusercontent.com/10134816/219375287-abb09594-137d-41e0-bc41-c6182661e650.png)

**Reply by xSunShine** [reply]
Any update on this  ? :)

**Reply by mgeeky** [reply]
But you know .IMG _is_ .ISO, right?

**Reply by xSunShine** [reply]
well i didn't know that,sorry then,since i somewhere before generated and file with img extention and it generated an image when open. Sorry then for all the fuss.
